src/mod2c_core/nocpout.c

```diff
@@ -3040,6 +3040,12 @@ sprintf(buf, "\
 \n  #pragma acc atomic capture\
 \n  _i = _nsb->_cnt++;\
 \n  if (_i >= _nsb->_size) {\
+\n#if defined(_OPENACC)\
+\n    int NetSendBuffer_t_not_large_enough = 0;\
```

Comment:
Just nitpick : avoid camel case? (net_send_buffer_not_large_enough)

Reply:
I actually started off with that, but then went with the camel case because the class name is in camel case. So, this choice is by design.